dolphinscheduler-ui/src/js/conf/home/router/index.js

```diff
@@ -397,6 +397,14 @@ const router = new Router({
           meta: {
             title: `${i18n.$t('Token manage')}`
           }
+        },
+        {
+          path: '/security/Alarm-plugin-example',
+          name: 'Alarm-plugin-example',
+          component: resolve => require(['../pages/security/pages/alarmPluginExample/index'], resolve),
```

Comment:
can you change `alarm` to `alert` ?

Reply:
Okay, because the local library is re-forked, merge first, and then I modify it together

Reply:
ok